sphinx/environment/__init__.py

```diff
@@ -96,10 +96,9 @@
 def sphinx_smartquotes_action(env):
     try:
```

Comment:
the try is needed because Docutils SmartQuotes does not have the ``smartquotes_action`` attribute at Docutils < 0.14.